Gateway/Request/DescriptionDataBuilder.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2021 Adyen NV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Gateway\Request;
+
+use Magento\Payment\Gateway\Data\PaymentDataObject;
+use Magento\Payment\Gateway\Helper\SubjectReader;
+use Magento\Payment\Gateway\Request\BuilderInterface;
+use Magento\Sales\Model\Order;
+
+class DescriptionDataBuilder implements BuilderInterface
+{
+    /**
+     * @param array $buildSubject
+     * @return array
+     */
+    public function build(array $buildSubject)
+    {
+        /** @var PaymentDataObject $paymentDataObject */
+        $paymentDataObject = SubjectReader::readPayment($buildSubject);
+        $payment = $paymentDataObject->getPayment();
+        /** @var Order $order */
+        $order = $payment->getOrder();
+
+        $request['body']['description'] = __('Order %1', $order->getIncrementId());
```

Comment:
This is shown in the payment page when the shopper clicks the link, should we add something more?

Reply:
@Aleffio what did we show in M1?

Reply:
we didn't have/fill `description` at the time: https://github.com/Adyen/adyen-magento/blob/a46ebd754912bb048cac3655325cf8a773790ffc/app/code/community/Adyen/Payment/Model/Api.php#L260-L283

Reply:
Thanks Alessio, I've added the website name and I think that's good for now